lua/lspconfig/denols.lua

```diff
@@ -79,7 +79,11 @@ local function denols_handler(err, method, result)
     end
   end
 
-  lsp.handlers[method](err, method, result)
+  if context.is_legacy_call then
+    lsp.handlers[context.method](err, context.method, result)
+  else
+    lsp.handlers[context.method](err, result, context)
+  end
```

Comment:
why can't you just do
```lua
util.compat_handler(lsp.handlers[method])(err, result, context)
```

Reply:
It's the same problem I mentioned earlier -- the wrapper works when you're making a new-style handler work with Neovim 0.5, but not the other way around. If the updated denols config is used with Neovim 0.5, the target handler will be using legacy arguments, so that wrapper won't do anything for it.

Reply:
Now I get it. Maybe add a `TODO` comment before it so it's easier to find and revert when the handler change is in a release?